src/pages/gd/Claim/Claim.tsx

```diff
@@ -0,0 +1,105 @@
+import { ClaimProvider, ClaimWizard } from '@gooddollar/good-design'
+import { NewsFeedProvider, SupportedChains } from '@gooddollar/web3sdk-v2'
+import { noop } from 'lodash'
+import { useEthers } from '@usedapp/core'
+import { useConnectWallet } from '@web3-onboard/react'
+import { Spinner, VStack } from 'native-base'
+import { useFeatureFlagWithPayload } from 'posthog-react-native'
+
+import { getNetworkEnv } from 'utils/env'
+
+import { feedConfig } from 'components/NewsFeed'
+
+import { PageLayout } from 'components/Layout/PageLayout'
+import { useDisabledClaimingModal } from './useDisabledClaimModal'
+
+const goodIdExplorerUrls = {
+    CELO: process.env.REACT_APP_GOODID_CELO_EXPLORER ?? 'https://api.celoscan.io/api?&',
+    FUSE: process.env.REACT_APP_GOODID_FUSE_EXPLORER ?? 'https://explorer.fuse.org/api?&',
+    MAINNET: process.env.REACT_APP_GOODID_MAINNET_EXPLORER ?? '',
+    GOODCOLLECTIVE: process.env.REACT_APP_GOOD_GOODCOLLECTIVE_EXPLORER ?? '',
+}
+
+const ClaimPage = () => {
+    const { account, chainId } = useEthers()
+    const [, connect] = useConnectWallet()
+    const history = useHistory()
+    const networkEnv = getNetworkEnv()
+    const [, payload] = useFeatureFlagWithPayload('claim-feature')
+    const { enabled: claimEnabled, disabledMessage = '' } = (payload as any) || {}
+    const { Dialog, showModal } = useDisabledClaimingModal(disabledMessage)
+
+    // const { tasks } = nxTasks
+
+    // const onClaimSuccess = useCallback(async () => {
+    //     // if (isEmpty(tasks)) {
+    //     //     return
+    //     // }
+    //     // const today = moment().format('YYYY-MM-DD')
+    //     // const shownTasksToday = await AsyncStorage.getItem('shownTasksToday')
+    //     // if (shownTasksToday === today) {
+    //     //     return
+    //     // }
+    //     // await AsyncStorage.setItem('shownTasksToday', today)
+    //     // show a task dialog
+    // }, [])
+
+    const handleConnect = useCallback(async () => {
+        if (claimEnabled) {
+            const state = await connect()
+
+            return !!state.length
+        } else {
+            showModal()
+        }
+        return false
+    }, [connect, claimEnabled])
+
+    const onUpgrade = () => history.push('/goodid')
+
+    const onNews = () => history.push('/news')
+
+    if (chainId === undefined) return <Spinner variant="page-loader" size="lg" />
+
+    return (
+        <PageLayout title="Claim" faqType="claim">
+            <Dialog />
+            <VStack maxWidth="500" margin="auto" mt={0}>
+                <ClaimProvider
+                    activePoolAddresses={
+                        process.env.REACT_APP_UBIPOOL_ADDRESSES
+                            ? JSON.parse(process.env.REACT_APP_UBIPOOL_ADDRESSES)
+                            : {}
+                    }
+                    explorerEndPoints={goodIdExplorerUrls}
+                    supportedChains={[SupportedChains.CELO, SupportedChains.FUSE]}
```

Comment:
should be defaults in claimprovider

Reply:
Why? if any changes, we have to release new packages every time before we update/release the dapp?

Reply:
defaults means they are not required to specify but can be overridden if required

Reply:
obviously.
done here: https://github.com/GoodDollar/GoodProtocolUI/pull/578/commits/cb5214fcd46f843dd2e70aab6423f9b1ba9bd8af